### PR TITLE
Vision & Requirement Updates

### PR DESCRIPTION
**Pull request by timothyb0912**
This pull request is going to hold the edits to the vision and requirement docs. It addressess the first task in #80 .

As it stands, I've included a third item (for falsifying causal graphs) to our deliverables for the project vision. If you see the doc, I've included my rationale for why we should have an explicit third deliverable. Let me know if any of you disagree with its inclusion!

Beyond this, let me know your thoughts / critiques / suggestions for the vision doc!
@hassanobeid1994, @bouzaghrane

### CODE REVIEW

File: references/organization/_03-Vision.md [hunk 1]
```diff
@@ -16,11 +16,26 @@ jupyter:
...
-We will build on Brathwaite’s work and review paper and demonstrate with steps how to incorporate and think about causal inference in the context of DCM. We will also demonstrate the value of such an approach by contrasting the outcome of this way of thinking to that of traditional demand modeling where little care is given to the causal structure. 
+We will demonstrate, using simulations and a case study, how to approach travel demand modeling while paying attention to both the causal structure of how people make their choices and the treatment assignment process.
+To do this, we will apply the causal inference workflow outlined in Brathwaite and Walker (2018), and we will augment the workflow with additional details and guidance from our experience's in this project.
+We will also demonstrate the value of such an approach by contrasting the outcome of this way of thinking to that of traditional demand modeling where little care is given to the underlying causal structure of the problem.
```
Comment (bouzaghrane) [inline references/organization/_03-Vision.md]: Can we be more specific on how we will demonstrate that in here? Or is it unnecessary?

File: references/organization/_03-Vision.md [hunk 2]
```diff
@@ -16,11 +16,26 @@ jupyter:
...
+The goals of the simulation will be to (1) demonstrate the step-by-step causal inference workflow, and (2) show the pitfalls of ignoring the treatment assignment mechanism.
+
+The second deliverable of our work will be a combined simulation study and case study.
+This effort will focus on the more complicated and realistic case that is typically faced by demand modelers, where we have latent confounders that affect the treatment assignment of our causal variables of interest.
+Taking a high-profile and recent approach to coping with latent confounding, we will apply the de-confounder technique of Wang and Blei (2019).
+Our application will show how directed acyclic graphs can help increase transparency about one's reasoning regarding the number of confounders, assumptions for which variables are confounded, and the models needed to estimate the causal effects.
+We will investigate, through simulation, the usefulness, and pitfalls/sensitivity of this approach, and we will demonstrate the technique in a travel demand  modelling applicaton with real world data.
```
Comment (bouzaghrane) [inline references/organization/_03-Vision.md]: Same comment about simulation from above.

File: references/organization/_03-Vision.md [hunk 3]
```diff
@@ -16,11 +16,26 @@ jupyter:
...
+Specifically, there will be three main deliverables in our work/presentation.
 
-The second part of the paper will focus on the more complicated and realistic case that is typically faced by demand modelers, where we have latent confounders that affect the selection process into our causal variables of interest. We will apply the de-confounder approach by Blei et al. and demonstrate how DAGs can help reason/increase transparency about the number of confounders, assumptions for which variables are confounded, and models needed to estimate the causal effects. We will also demonstrate this approach using simulations, and quantify the sensitivity of this approach under different assumptions of variables confounded, number of confounders, and DAGs. Apply this approach to a real world dataset as well.
+The first deliverable is a simple application of the steps outlined in Brathwaite and Walker (2018) to a simulation example.
```
Comment (bouzaghrane) [inline references/organization/_03-Vision.md]: Should we specify what the simulation is based on? aka what dataset?

File: references/organization/_03-Vision.md [hunk 4]
```diff
@@ -16,11 +16,26 @@ jupyter:
...
+The goals of the simulation will be to (1) demonstrate the step-by-step causal inference workflow, and (2) show the pitfalls of ignoring the treatment assignment mechanism.
+
+The second deliverable of our work will be a combined simulation study and case study.
+This effort will focus on the more complicated and realistic case that is typically faced by demand modelers, where we have latent confounders that affect the treatment assignment of our causal variables of interest.
+Taking a high-profile and recent approach to coping with latent confounding, we will apply the de-confounder technique of Wang and Blei (2019).
+Our application will show how directed acyclic graphs can help increase transparency about one's reasoning regarding the number of confounders, assumptions for which variables are confounded, and the models needed to estimate the causal effects.
+We will investigate, through simulation, the usefulness, and pitfalls/sensitivity of this approach, and we will demonstrate the technique in a travel demand  modelling applicaton with real world data.
+
+The final deliverable of our work will be an empirical demonstration of methods for falsifying one's causal graph.
```
Comment (bouzaghrane) [inline references/organization/_03-Vision.md]: Based on a simulation study or real world data?

**Review by bouzaghrane — COMMENTED** [review]
This all looks good to me @timothyb0912 , very very minor comments.

**Comment by timothyb0912** [comment]
@bouzaghrane let me know if the changes addressed your comments!

**Review by bouzaghrane — APPROVED** [review]
Everything looks good to me!

**Comment by timothyb0912** [comment]
Alright, I think I'm done with a first revised draft of the requirements doc.

Can everyone take a look and leave a review with their:
- comments on what looks incorrect
- questions about what is required vs what has been completed
- questions for clarification.

Given that it's almost 2pm, I clearly don't expect such a review before our 3pm meeting (though that would be amazing).

@bouzaghrane, @hassanobeid1994

**Pull request merged**